elastic/kibana
# [Discover] Fix saved search with custom label running into "aborted" error on a dashboard

## Summary

This PR takes care of saved searches running into the aborted state when initially rendered on a Dashboard. This happens when there's a custom title was assigned to the saved search. This triggered a reload causing the current request to be aborted.

### Testing
* This bug is just reliable reproduceable in a cloud instance
* Add a saved search on a dashboard, and assign a custom title to it
* Reload the dashboard. it should work like expected

Before it looked like this:

![](https://user-images.githubusercontent.com/22050363/202500875-e6c197f0-01ff-48c0-924b-5fe5da3f4006.png)

Fixes #145589


### Checklist
